book/4 - MultiWorlds/1 - Multiworlds Overview.html

```diff
@@ -0,0 +1,19 @@
+<p>
+	Cuberite supports multiple worlds. Each world has its own `world.ini` file. Additional Worlds can be added by editing <code>settings.ini</code>. This is explained in the example below.
+</p>
+
+<figure class="codebox">
+<pre><code>[Worlds]
+DefaultWorld=world
+World=myNewWorld
+World=HappyLand
+</code></pre>
+</figure>
+
+<p>
+	In the example above, 2 extra worlds are added to the server. Note that this automatically creates 2 additional configuration files, namely <code>myNewWorld/world.ini</code> and <code>HappyLand/world.ini</code>.
+</p>
+
+<p>
+	The rest of this section deals with linking your worlds and traveling between them.
```

Comment:
linking your worlds together?

Also, is the manual BrE or AmE style? If BrE then it should be travelling.